src/dmd/dmangle.d

```diff
@@ -1171,6 +1204,7 @@ extern (C++) const(char)* mangleExact(FuncDeclaration fd)
         OutBuffer buf;
         scope Mangler v = new Mangler(&buf);
         v.mangleExact(fd);
+        v.__dtor();
```

Comment:
Give it a proper name, rather than calling an automatic name (See https://github.com/dlang/dmd/pull/11154)